### ISSUE TITLE
Cannot trigger Philips Hue light effects on Philips Hue light groups

### ISSUE THREAD

**Issue by Grandma-Betty** [Koenkk/zigbee2mqtt#23978]
### What happened?

The following issue was closed a long time ago but the issues still persists as other users reported too:
https://github.com/Koenkk/zigbee2mqtt/issues/15997#issuecomment-2350208617

Therefore I will open a new issue for this issue as it seems the other ticket is not being followed by any developer anmore.

To sum it up, here is a description of the issue:
On Home Assistant, when triggering any Philips Hue light effects on a single Philips Hue light entity, the according light effect is being triggered without any issues.
So far, so good.
On the other hand, when the same Philips Hue light entities are being grouped together and then triggering the same specific Philips Hue light effects on that light group, the according light effects do not apply.

Please be aware that _some_ specific Philips Hue light effects actually do indeed work when being triggered on Philips Hue light groups without any issues, while others do not.

I have tested with the following two lights grouped together:

- Philips Hue Festavia Light String 20m ([Model](https://www.zigbee2mqtt.io/devices/9290036744.html#philips-9290036744))
- Philips Hue Festavia Light String 40m ([Model](https://www.zigbee2mqtt.io/devices/929003674601.html#philips-929003674601))

The following Philips Hue light effects **are working** on that group:
- blink
- breathe
- okay
- channel_change

The following Philips Hue light effects **are not working** on that group:
- candle
- fireplace
- colorloop
- sunrise
- sparkle
- opal
- glisten
- finish_effect
- stop_effect
- stop_hue_effect

### What did you expect to happen?

Philips Hue light effects should always apply on Philips Hue lights, whether being triggered on a single Z2M light entity or on a Z2M light group entity.

### How to reproduce it (minimal and precise)

Group Philips Hue light entities together and then trigger the according Philips Hue light effects on that group.

### Zigbee2MQTT version

1.40.1

### Adapter firmware version

20221226

### Adapter

Sonoff zStack3x0

### Setup

Add-on on virtual HAOS x86_64 (VMware ESXi)

**Comment by deviantintegral** [comment]
I think this is a limitation in the Hue firmware, though I haven’t confirmed that myself.

I will say, I used to think that preferring Zigbee groups was always the best way to set up a network. However, I discovered that on my own network with 45 routers and 64 end devices that reducing the number of groups I used really helped with overall network performance.

For example, I used to have a group in a bathroom fixture that had 2 lights in it. When I sent that group a command, the routers would repeat the command several times as broadcasts. And, with adaptive lighting, there were commands being sent periodically to adjust the light temperature.

What I ended up doing was moving all of my small groups to Home Assistant groups. Surprisingly, I rarely notice any popcorn effects as lights in an HA group come on, which I did years ago when first setting things up. I still have Zigbee groups for groups like “All lights”, “Main floor”, and so on, since those would be sending so many commands that the broadcast storm doesn’t really matter.

And, one side effect is that sending hue light effects always works (because I would never trigger them on a whole floor or the whole house anyways).
